digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/linearasset/NumericValueLinearAssetService.scala

```diff
@@ -22,7 +22,7 @@ class NumericValueLinearAssetService(roadLinkServiceImpl: RoadLinkService, event
   override def eventBus: DigiroadEventBus = eventBusImpl
   override def polygonTools: PolygonTools = new PolygonTools()
   override def assetDao: PostGISAssetDao = new PostGISAssetDao
-
+  def createTimeStamp(i:Int=5): Long = LinearAssetUtils.createTimeStamp(i) 
```

Comment:
Tässä olis varmaan hyvä olla joku yhtenäisyys, et luotais vastaava funktio kaikkiin luokkiin, missä käytetään, tai sit käytettäis kaikissa suoraan tuolta utilsista.

Reply:
hyym ehkä, tää oli vaan ovverridaamista varten tuolla testissä

Reply:
mut yhtenäisyys varmaan ihan hyvä